tests/hash-backend-integration/src/tests/util.ts

```diff
@@ -1,3 +1,6 @@
+/** @todo: figure out why omitting this import causes a TSC error */
+import "@apps/hash-api/src/express";
```

Comment:
@CiaranMn I don't understand why this was caused in this PR, but the `@tests/hash-backend-integration` TSC check was failing because it didn't pick up on the overridden `Express` definition. Explicitly importing it was the only way I could think of to resolve this.

Reply:
Thanks, should've thought of that, have addressed in https://github.com/hashintel/hash/pull/3929/commits/f30b1e786919b4970474da8207bec85cfae6349b. I'm going to keep `express.d.ts` because it allows us to get rid of the eslint ignore and it probably should be in a decleration file anyway.